dask_sql/context.py

```diff
@@ -250,22 +250,20 @@ def create_table(
             **kwargs,
         )
 
-        self.schema[schema_name].tables[table_name.lower()] = dc
-
-        if statistics:
-            self.schema[schema_name].statistics[table_name.lower()] = statistics
-        elif parquet_statistics:
+        if parquet_statistics and not statistics:
             statistics = parquet_statistics(dc.df)
             if statistics:
                 row_count = 0
                 for d in statistics:
                     row_count += d["num-rows"]
                 statistics = Statistics(row_count)
-                self.schema[schema_name].statistics[table_name.lower()] = statistics
-
         # If no statistics are obtainable, we will just assume 100 rows
         if not statistics:
             statistics = Statistics(100)
```

Comment:
I'd prefer having something like N/A / None. Since we don't want to introduce weird join re-ordering situations for cases where the stats were not available by assuming this was `100/0` rows. 

That being said if changing this to

Reply:
Thanks, makes sense to me!